pkg/cmd/pipeline/start.go

```diff
@@ -692,3 +698,103 @@ func parsePipeline(taskLocation string, p cli.Params) (*v1alpha1.Pipeline, error
 	}
 	return &pipeline, nil
 }
+
+func (opt *startOptions) getInputWorkspaces(pipeline *v1alpha1.Pipeline) error {
+	for _, ws := range pipeline.Spec.Workspaces {
+		fmt.Fprintf(opt.stream.Out, "Please give specifications for the workspace: %s \n", ws.Name)
+		name, err := askParam("Name for the workspace :", opt.askOpts)
+		if err != nil {
+			return err
+		}
+		workspace := "name=" + name
+		subPath, err := askParam("Value of the Sub Path :", opt.askOpts, " ")
+		if err != nil {
+			return err
+		}
+		if subPath != " " {
+			workspace = workspace + ",subPath=" + subPath
+		}
+
+		kind, err := askParam("Type of the workspace :", opt.askOpts, "emtpyDir")
```

Comment:
Should we give it a list (`emptyDir`, `config`, …) and then an `option` part where the user can specify the rest of the option, *per* type ?